src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Typography } from '../Typography';
+import { DivStyled } from './Badge.styles';
+import { BadgeProps } from './types';
+
+const Badge: React.FC<BadgeProps> = ({
+    text,
+    state = 'normal',
+    textVariant = 'body16',
+    ...otherProps
+}) => {
+    return (
+        <DivStyled state={state} data-testid="test-badge-id" {...otherProps}>
```

Comment:
We do not want to allow other props to be passed

Reply:
Removed it.